crates/ark/src/help_proxy.rs

```diff
@@ -128,8 +132,14 @@ async fn proxy_request(req: HttpRequest, app_state: web::Data<AppState>) -> Http
     // Add query from original request back to URL.
     target_url.set_query(Some(query));
 
+    // Set up reqwest client with 3 retries.
+    let retry_policy = ExponentialBackoff::builder().build_with_max_retries(3);
```

Comment:
For the curious, here are the other default options that get set for the backoff:

https://docs.rs/retry-policies/0.4.0/src/retry_policies/policies/exponential_backoff.rs.html#148-151

Definitions
https://docs.rs/reqwest-retry/latest/reqwest_retry/policies/struct.ExponentialBackoff.html#fields